src/com/dabomstew/pkrandom/constants/Gen3Constants.java

```diff
@@ -236,7 +239,62 @@
                 mysticWater, sharpBeak, poisonBarb, neverMeltIce, spellTag, twistedSpoon, charcoal, dragonFang,
                 silkScarf, shellBell, seaIncense, laxIncense));
         list.addAll(consumableHeldItems);
-        return list;
+        return Collections.unmodifiableList(list);
+    }
+
+    public static final List<Integer> generalPurposeConsumableItems = Collections.unmodifiableList(Arrays.asList(
+            cheriBerry, chestoBerry, pechaBerry, rawstBerry, aspearBerry, leppaBerry,
+            oranBerry, persimBerry, lumBerry, sitrusBerry, ganlonBerry, salacBerry,
+            // An NPC pokemon's nature is generated randomly with IVs during gameplay. Therefore, we include
+            // all these berries, even though they might confuse the opponent's pokemon if the nature is wrong.
+            figyBerry, wikiBerry, magoBerry, aguavBerry, iapapaBerry,
+            apicotBerry, lansatBerry, starfBerry, berryJuice, whiteHerb, mentalHerb
+    ));
+
+    public static final List<Integer> generalPurposeItems = Collections.unmodifiableList(Arrays.asList(
+            brightPowder, quickClaw, choiceBand, kingsRock, focusBand,
```

Comment:
Choice Band shouldn't be here, since you add it separately in Gen3RomHandler if a move is physical

Reply:
Done.